<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1425</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Таблица учёта</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1431</width>
      <height>731</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1425</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>файл</string>
    </property>
    <addaction name="info_action"/>
    <addaction name="separator"/>
    <addaction name="new_action"/>
    <addaction name="open_action"/>
    <addaction name="save_action"/>
    <addaction name="separator"/>
    <addaction name="exit_action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>операции</string>
    </property>
    <addaction name="separator"/>
    <addaction name="change_action"/>
    <addaction name="add_action"/>
    <addaction name="delete_action"/>
    <addaction name="separator"/>
    <addaction name="search_action"/>
    <addaction name="reset_action"/>
    <addaction name="separator"/>
    <addaction name="day_action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="info_action">
   <property name="text">
    <string>справка</string>
   </property>
  </action>
  <action name="open_action">
   <property name="text">
    <string>открыть</string>
   </property>
  </action>
  <action name="save_action">
   <property name="text">
    <string>сохранить</string>
   </property>
  </action>
  <action name="exit_action">
   <property name="text">
    <string>выход</string>
   </property>
  </action>
  <action name="add_action">
   <property name="text">
    <string>добавить</string>
   </property>
  </action>
  <action name="delete_action">
   <property name="text">
    <string>удалить</string>
   </property>
  </action>
  <action name="search_action">
   <property name="text">
    <string>поиск</string>
   </property>
  </action>
  <action name="day_action">
   <property name="text">
    <string>день</string>
   </property>
  </action>
  <action name="new_action">
   <property name="text">
    <string>новый</string>
   </property>
  </action>
  <action name="change_action">
   <property name="text">
    <string>изменить</string>
   </property>
  </action>
  <action name="reset_action">
   <property name="text">
    <string>сброс поиска</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
